### ISSUE TITLE
Make updates faster

### ISSUE THREAD

**Issue by KapJI** [leikoilja/ha-google-home#60]
## Problem

Updates now are unreasonably slow and take around 20 seconds.

## Solution
- Run zeroconf only for setup
- Reuse tokens and don't refresh them for every update, they should be valid at least for 1 hour.
- Query alarms and timers from devices in parallel (probably already done)

Basically we can always assume that token is still valid and if request fails, only then we need to fetch the new one and retry the request.

With this update interval can be pretty small like 10 or 15 seconds and there is no need in #58
